tests/src/test/scala/com/nvidia/spark/rapids/CastOpSuite.scala

```diff
@@ -1123,6 +1123,8 @@ class CastOpSuite extends GpuExpressionTestSuite {
 
 /** Data shared between CastOpSuite and AnsiCastOpSuite. */
 object CastOpSuite {
+  val SPARK_OVERFLOW_INSERT_CLASS_NAME =
+    "org.apache.spark.sql.catalyst.expressions.CheckOverflowInTableInsert"
```

Comment:
Why is this declared here when it's used in AnsiCastOpSuite?  Why not put it where it's needed?